Userland/Applications/Spreadsheet/SpreadsheetView.cpp

```diff
@@ -228,6 +228,28 @@ SpreadsheetView::SpreadsheetView(Sheet& sheet)
         m_table_view->move_cursor(GUI::AbstractView::CursorMovement::Down, GUI::AbstractView::SelectionUpdate::Set);
     };
 
+    m_table_view->row_header().set_context_menu_policy(GUI::ContextMenuPolicy::CustomContextMenu);
+    m_table_view->row_header().on_custom_menu_requested = [this](GUI::ContextMenuEvent& event) {
```

Comment:
This doesn't feel very intuitive, on two basis:
- It's pretty odd to select a row when its header is _right-clicked_
- Once I set an option for a row like this, I feel like I expect it to apply to further cells created in that row.

I'm not sure what the right way to do this is, but this doesn't feel right.

Reply:
Selecting the row on right click I think is required based on the fact that whatever operation you'll be selecting from the context menu would apply to all cells in the row.

I agree though about needing to select additional cells when the row is selected

Reply:
@alimpfard went ahead and made the requested changes